doc/bash_autocompletion.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+if [ -z $BASH ]; then
```

Comment:
Thats wrong. If BASH is empty it will result on an invalid expression. Quote the var with ""